### PR TITLE
Inject version key in composer file on zip creation

### PR DESCRIPTION
**Pull request by comzeradd**
This is just the counterpart of greenpeace/planet4-master-theme#1392

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -325,19 +325,28 @@ jobs:
...
+          name: Remove unnecessary files
           command: rm -rf .circleci .git .githooks assets/src bin tests
-      # Exclude node_modules instead of removing, which takes a long time (lots of small files).
       - run:
-          name: "Create zip file"
+          name: Inject version to composer
+          command: |
+            apk add jq
+            cat composer.json | jq --arg version ${CIRCLE_TAG} '. + {version: $version}' > tmp.json
+            cp -f tmp.json composer.json
```
Comment (lithrel) [inline .circleci/config.yml:344]: I just realized tmp.json ends up in the final zip

**Review by lithrel — APPROVED** [review]
:+1:

File: .circleci/config.yml (hunk 1 above)
Comment (Inwerpsel) [reply .circleci/config.yml:344]: If it gets immediately overwritten in the next statement, we could also do 
```bash
cat composer.json | jq --arg version ${CIRCLE_TAG} '. + {version: $version}' > composer.json

Comment (Inwerpsel) [reply .circleci/config.yml:344]: Using `mv tmp.json composer.json` would fix that

**Pull request merged**